src/develop/blend.c

```diff
@@ -2644,7 +2644,7 @@ void dt_develop_blend_process(struct dt_iop_module_t *self, struct dt_dev_pixelp
     }
 
 #ifdef _OPENMP
-#if !defined(__SUNOS__) && !defined(__NetBSD__) && !defined(__WIN32__)
+#if !defined(__SUNOS__) && !defined(__NetBSD__) && !defined(__WIN32__) && defined(__GLIBC__)
```

Comment:
As best I'm aware, this block was working fine with the various flavours of linux that the core team use. Those flavours all use glib, and if they had a problem with that block I would expect them to note it and fix it.